### PR TITLE
mgr/dashboard: Improve str_to_bool

### PR DESCRIPTION
**Pull request by votdev**
Signed-off-by: Volker Theile <vtheile@suse.com>

### CODE REVIEW

File: src/pybind/mgr/dashboard/tools.py [hunk 1]
```diff
@@ -744,4 +745,4 @@ def getargspec(func):
 def str_to_bool(var):
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard/tools.py]: could you add a docstring like this?:
```python
"""
>>> str_to_bool('true') and str_to_bool('yes') and str_to_bool('1') 
True
"""
```
and make sure it's tested by tox?

Comment (votdev) [reply src/pybind/mgr/dashboard/tools.py]: Done

**Comment by s0nea** [comment]
Hey @votdev do we need to run the Teuthology tests on this PR?

**Comment by sebastian-philipp** [comment]
+1 for running Teuthology.

**Comment by s0nea** [comment]
QA test run was successful:
- https://shaman.ceph.com/builds/ceph/wip-tdehler-testing-22757/
- http://pulpito.ceph.com/tdehler-2018-07-03_13:45:25-rados:mgr-wip-tdehler-testing-22757-distro-basic-smithi/

**Pull request merged**